### ISSUE TITLE
Line Jump option is missing by default

### ISSUE THREAD

**Issue by tmal-1** [jgraph/drawio-desktop#1442]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**
The Line Jump option is missing by default. If I change the line type (Waypoints) without actually changing it (change a straight line to a straight line) the missing option appears.

**To Reproduce**
1. Select a line element within the drawing
2. Notice in the Style tab that Waypoints are Straight by default
3. Click on Waypoints and select Straight (a change that shouldn't be a change)
4. Notice that the Style tab now contains a Line Jumps row

**Expected behavior**
Steps 2 and 3 shouldn't be necessary. When you select a line element (1) the Style tab should contain a Line Jumps row (4).

**Screenshots**
<img width="246" alt="Screenshot 2023-08-23 at 16 18 09" src="https://github.com/jgraph/drawio-desktop/assets/122500060/aa7a971c-e94b-4fd4-98be-220a6acd8948">
<img width="242" alt="Screenshot 2023-08-23 at 16 18 34" src="https://github.com/jgraph/drawio-desktop/assets/122500060/d91021b1-94c5-4495-8064-cb6e89f522e8">

**draw.io version (In the Help->About menu of the draw.io editor):**
- draw.io version 21.6.8

**Desktop (please complete the following information):**
 - OS: MacOS Ventura 13.4.1 (c) 

**Additional context**

**Comment by m-mohamedin** [comment]
Unfortunately, we cannot reproduce. Is it still occurring in the latest release?

**Comment by tmal-1** [comment]
I have re-tested this using v22.1.2 and v22.1.15, which I believe is the latest version. Both still exhibit this behaviour.

When you attempted to reproduce this, did you start from a newly-opened instance of draw.io?

1. If draw.io was already running, Quit (Command-Q).
2. Open the draw.io application.
3. Create New Diagram -> Blank Diagram -> Create
4. Insert a line. The Style tab now resembles my first screenshot - i.e. the 'Line jumps' option is missing.
5. Click on Waypoints and select Straight. The Style tab now resembles my second screenshot.

After the 'Line jumps' option appears it will remain visible for subsequent lines which is why you need to start from a newly-opened instance.

**Comment by tmal-1** [comment]
For completeness I have also tested this with the web version of draw.io (also v22.1.15), which does _not_ exhibit this behaviour.

**Comment by m-mohamedin** [comment]
![Screenshot 2023-12-29 at 2 51 05 PM](https://github.com/jgraph/drawio-desktop/assets/24944765/482ff488-3a36-4816-b388-461d282314cf)
This is what I see in the latest version of the desktop (the line I inserted, is from the "General" shapes)

**Comment by tmal-1** [comment]
Did you quit the application before carrying out this test? The application exhibits this behaviour when newly started but, after the Line Jumps option is visible it remains visible until the application is restarted.

Are you running the same version of MacOS?

I have identified three different versions of the application that exhibit this behaviour.

**Comment by m-mohamedin** [comment]
Yes, I tried on MacOS (latest version) and quit the app before testing. Maybe a video can make it clearer, please record a video showing the steps.
Thanks

**Comment by tmal-1** [comment]
I have uploaded a video as requested. This is smaller than I would like, to meet a GitHub limit on file sizes.

https://github.com/jgraph/drawio-desktop/assets/122500060/f870a6cc-87b2-4c65-b701-b35d1d45f455

Notice this is v22.1.18, which is more recent than the versions I originally reported. So I can confirm that several versions exhibit this behaviour:
* v21.6.8
* v22.1.2
* v22.1.15
* v22.1.18

**Comment by brunogauthier1** [comment]
Same problem here, I created my schema, probably before the line jump feature was available in draw.io.
If I create a new line in the schema, the "line jump" menu feature appears when selecting this new line.
However, the same "line jump" menu feature is absent on the previously created lines. 
I'm running on chrome + Ubuntu 22.04 LTS

**Comment by alderg** [comment]
What are the styles for the old and new lines (Ctrl+E)?

**Comment by tmal-1** [comment]
> What are the styles for the old and new lines (Ctrl+E)?

If I create a line immediately after drawio opens the style is:
endArrow=none;html=1;rounded=0;fontSize=12;startSize=8;endSize=8;curved=1;

If I repeat after selecting straight waypoints the style is:
endArrow=none;html=1;rounded=0;fontSize=12;startSize=8;endSize=8;

That suggests 'curved=1' is the default setting for waypoints.

This made me realise a different way of explaining the bug. If I select curved waypoints then the line jumps options disappear.If you are tempted to suggest that curved waypoints do not support line jumps I would point out that there appears to be a user interface bug that shows straight waypoints in the Style tab even though the default line style has curved waypoints.

I have now installed drawio v27.0.9 and can confirm that it still exhibits this behaviour.

**Comment by davidjgraph** [comment]
Curved lines don't support line jumps.

**Comment by tmal-1** [comment]
I had already anticipated your reply: “ If you are tempted to suggest that curved waypoints do not support line jumps I would point out that there appears to be a user interface bug that shows straight waypoints in the Style tab even though the default line style has curved waypoints.”

I don’t expect curved lines to support line jumps. I explained a user interface bug when curved waypoints are enabled.

**Comment by tmal-1** [comment]
Why did you delete a comment that both explained why your reply was not relevant and explained a remaining bug?

**Comment by tmal-1** [comment]
Incorrectly closed.

The bug is that curved waypoints display incorrectly in the status as straight waypoints. That is not to suggest that curved waypoints should support line jumps.

**Comment by alderg** [comment]
You're right, sorry for the confusion. Choosing Straight in the Waypoints menu also removed the curved line style, so picking the option that was shown changed the line. In the next release, Straight keeps the Line style (sharp, rounded or curved), so the two settings are independent. Curved lines still don't support line jumps.
